ngraph/core/src/op/topk.cpp

```diff
@@ -508,7 +508,10 @@ void op::v1::TopK::validate_and_infer_types()
             auto max_k = maximum_value(input_value(1));
             if (max_k.first)
             {
-                output_shape[m_normalized_axis] &= Dimension(0, max_k.second);
+                const auto in_max = input_partial_shape[m_normalized_axis].get_max_length();
```

Comment:
Should it be `get_min_length` for the lower bound and `get_max_length` for the upper bound?

Reply:
Yes, I'll correct that.